contracts/JBTokenStore.sol

```diff
@@ -265,8 +285,9 @@ contract JBTokenStore is JBControllerUtility, JBOperatable, IJBTokenStore {
     bool _preferClaimedTokens
   ) external override onlyController(_projectId) {
     // An amount must be specified.
-    require(_amount > 0, '0x22: NO_OP');
-
```

Comment:
super nit) let's leave this space

Reply:
@odd-amphora the space is there below  the if statement, https://github.com/viraj124/juice-contracts-v2/blob/main/contracts/JBTokenStore.sol#L245 if that's what you meant